scripts/next-data/generatePreBuildFiles.mjs

```diff
@@ -29,6 +34,25 @@ export const generateBlogYearPages = cachedBlogData =>
     .then(data => [...new Set(data)])
     .then(data => data.forEach(createBlogYearFile));
 
+/**
+ * Render markdown to html via mdx
+ * @param {string} body
+ * @returns {Promise<string>}
+ * @see {https://github.com/mdx-js/mdx/discussions/1985}
+ */
+export const renderMarkdown = async body => {
+  const { default: mdx } = await evaluate(body, {
```

Comment:
Hmm, afaik we already should have a rendered version of this at the time, no?

Reply:
Where exists renderred version?
`post` object just have file path…

```js
{
  post: {
    title: 'Node v19.5.0 (Current)',
    author: 'Rafael Gonzaga',
    date: 2023-01-24T22:12:56.567Z,
    category: 'release',
    slug: '/blog/release/v19.5.0',
    file: 'v19.5.0.md'
  }
}
```

Reply:
I think this is just what we're passing as a prop, you should check the parent functions, as I think I manually build this "post" object.